caravel/assets/javascripts/SqlLab/components/QueryHistory.jsx

```diff
@@ -7,6 +7,16 @@ import * as Actions from '../actions';
 import QueryTable from './QueryTable';
 import { Alert } from 'react-bootstrap';
 
+const propTypes = {
+  queries: React.PropTypes.object,
+  tabHistory: React.PropTypes.array,
```

Comment:
we should either make `tabHistory` and `actions` required, or add default props. 
`tabHistory: React.PropTypes.array.isRequired`,